### DISCUSSION TITLE
"Models" in the sidebar should be modifiable

### DISCUSSION THREAD

**Discussion by coin-au-carre** [scalar/scalar#2246] [Ideas]
In our API spec we have a tag named "Models" and this is colliding with the harcoded "Models" which represent the component schemas: 
https://github.com/scalar/scalar/blob/e88a2c2f64e51c843b1a7a7a703309029792284e/packages/api-reference/src/hooks/useSidebar.ts#L182 

![image](https://github.com/scalar/scalar/assets/1044719/9aa34583-73da-4c9e-8a2a-eb9aa5ea92ce)

It would be nice to be able to modify naming of "Models" as input parameter.

**Comment by hanspagel** [comment]
Ah, interesting! I think you can overwrite with some custom CSS. Did you try that?

**Reply by coin-au-carre** [reply]
Thanks for the reply I have not tried that yet. Can you suggest a reference, is there a doc for that ?
